apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -432,10 +432,12 @@ handle_request_('PostTransaction', #{'Tx' := Tx}, _Context) ->
                         ok ->
                             Hash = aetx_sign:hash(SignedTx),
                             {200, [], #{<<"tx_hash">> => aehttp_api_encoder:encode(tx_hash, Hash)}};
-                        {error, _} ->
+                        {error, Reason} ->
+                            lager:debug("Invalid tx: ~p", [Reason]),
```

Comment:
Actually I see similar message already logged in [`aec_tx_pool:check_pool_db_put`](https://github.com/aeternity/aeternity/blob/831e1ee30e4092058ca35573bdfd8dd5f392c01d/apps/aecore/src/aec_tx_pool.erl#L665-L667).